Building with make fails

Building unik with `make` fails with this error message:

```
pkg/daemon/daemon.go:16: imported and not used: "runtime"
make: *** [localbuild] Error 2
Makefile:254: recipe for target 'localbuild' failed
make: *** [Makefile:247: binary] Error 2
```

OS: Arch Linux

--- a/pkg/daemon/daemon.go
+++ b/pkg/daemon/daemon.go
@@ -13,7 +13,6 @@ import (
 	"time"
 
 	"path/filepath"
-	"runtime"
 
 	"github.com/Sirupsen/logrus"
 	"github.com/docker/docker/pkg/ioutils"

